src/Symfony/Bundle/WebServerBundle/WebServerConfig.php

```diff
@@ -33,6 +33,7 @@ public function __construct($documentRoot, $env, $address = null, $router = null
         }
 
         putenv('APP_FRONT_CONTROLLER='.$file);
```

Comment:
Is it possible to just remove `putenv` here?

Reply:
I think we should keep it because like @ostrolucky said there could be cases where $_ENV could be empty

Reply:
I also think we should remove the call to putenv (but alongside with -dvariable_order)
and then, symfony/process should be bumped to symfony/process `^3.4.2` in the bundle's composer.json